symengine/ruby/valgrind_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+valgrind --tool=memcheck --num-callers=15 --partial-loads-ok=yes --undef-value-errors=no ruby -S rspec spec
```

Comment:
The same comment about the other script.

Also, you still need to document how this script is meant to be used.